libraries/Microsoft.BotKit/BotkitConversationState.cs

```diff
@@ -15,15 +15,24 @@ namespace Microsoft.BotKit
     /// </summary>
     public class BotkitConversationState : ConversationState
     {
-        public BotkitConversationState(IStorage storage) : base(storage)
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BotkitConversationState"/> class.
+        /// </summary>
+        /// <param name="storage">BotkitConversationState constructor.</param>
```

Comment:
This description doesn't belong to the `storage` paremeter

Reply:
Done